Dwolla.Client/Models/Responses/Customer.cs

```diff
@@ -22,7 +22,12 @@ public class Customer
         public string State { get; set; }
         public string PostalCode { get; set; }
         public string Phone { get; set; }
+        public string DateOfBirth { get; set; }
+        public string Ssn { get; set; }
         public string BusinessName { get; set; }
+        public string BusinessClassification { get; set; }
+        public string BusinessType { get; set; }
+        public string Ein { get; set; }
```

Comment:
These actually only make sense and are already included in CreateCustomerRequest. They're not returned by the GET customers/:id or GET customers endpoints. I'll remove the properties I added in 6570db7.

Reply:
Ok. Are the fields listed at https://docsv2.dwolla.com/#retrieve-a-customer the only fields returned in all statuses?

Reply:
It looks like we do return the fields I added in f83ba0e if it's a VCR Personal or a VCR Business so I'm leaving those. However, we do not return the fields added in this PR at this time.